samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace Azure.AI.DocumentTranslation
 {
+    // Data plane generated client. The DocumentTranslation service client.
```

Comment:
Why we not put it in the `<summary>`? And why not 
```suggestion
    /// Data plane generated client. The DocumentTranslation service client.
```

Reply:
> Why we not put it in the `<summary>`? And why not

@pshao25 see the comments from Ted: https://github.com/Azure/azure-sdk-for-net/pull/29041#issuecomment-1142799105